react-backend/routes/activities.js

```diff
@@ -1,15 +1,24 @@
 const express = require('express');
 
+const jwt = require('jsonwebtoken');
+
 const router = express.Router();
 
 const activities = require('../database/queries/activities');
 
 router.get('/', (req, res) => {
-  activities()
-    .then(activities => res.send({ activities }))
-    .catch(err => {
+  jwt.verify(req.headers.authorization, process.env.SECRET, (err, payload) => {
+    if (err) {
       console.log(err);
-    });
+      res.send(JSON.stringify({ error: 'Not logged in' }));
+    } else {
+      activities()
+        .then(activities => res.send({ activities }))
+        .catch((err) => {
+          console.log(err);
```

Comment:
as mentioned in `getAllUsers.js` we need to send a 500 error to the front end in this error case.

Reply:
added res.status(500).send(err) if it's ok to do error handling like this?